Zend/tests/attributes/deprecated/class_constants/reflection_001.phpt

```diff
@@ -0,0 +1,17 @@
+--TEST--
+#[\Deprecated]: ReflectionClassConstant::isDeprecated() returns true.
```

Comment:
Reflection tests should probably go in ext/reflection.

Reply:
There's precedent of reflection-based tests within `Zend/`. I would prefer keeping all the `#[\Deprecated]` tests within a single place instead of distributing it all over the source-tree, but can move them if you insist.

Reply:
In principle, I agree. The distinction for reflection, standard and other required extensions is blurry. But we are still relatively consistent on this, I believe.